global.php

```diff
@@ -206,7 +206,7 @@
 	// If we're accessing poll results, fetch the forum theme for it and if we're overriding it
 	else if(isset($mybb->input['pid']) && THIS_SCRIPT == "polls.php")
 	{
-		$query = $db->simple_select('threads', 'fid', "poll = '{$mybb->input['pid']}'", array('limit' => 1));
+		$query = $db->query("SELECT t.fid FROM ".TABLE_PREFIX."polls p LEFT JOIN ".TABLE_PREFIX."threads t ON (t.tid=p.tid) WHERE p.pid = '{$mybb->input['pid']}' LIMIT 1");
```

Comment:
Shouldn't this be an `INNER JOIN` rather than a `LEFT JOIN`?

Reply:
So make it INNER JOIN if you think that's better.

Reply:
Ok. Not everybody likes to have their PRs changed without prior discussion or warning.

Reply:
Also it looks like this PR isn't commits from maintainers. If you enable this, I can edit this PR directly so that it shows up in the log correctly: https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
PR is from Xerotic so he will have to do the the configuration change.

Reply:
Please do it )